projects/packages/forms/src/blocks/contact-form/edit.js

```diff
@@ -327,7 +331,7 @@ export const JetpackContactFormEdit = forwardRef(
 							instanceId={ instanceId }
 						/>
 					) }
-					{ ! isSimpleSite() && (
+					{ ! ( isSimpleSite() || isAtomicSite() ) && (
```

Comment:
It looks like I wrapper too much onto check? Jetpack CRM should stay.

And prolly keep showing Creative Mail if it was already configured.

Reply:
I think that's fine for now, to be honest. It makes sense to me to keep things simpler on WoA. But you could certainly improve the logic in a follow-up PR to keep promoting Jetpack CRM.

Reply:
👍 

Ideally we also promote on simple, and just put them through business plan upgrade.